packages/design-system/src/components/ButtonGroup/ButtonGroup.md

```diff
@@ -25,19 +25,25 @@ By default ButtonGroup use internal state to track currently selected button. It
 If you want to use ButtonGroups as controlled component you can set `currentIndex` props to set which button is active.
 
 ```js
-<ButtonGroup currentIndex={1}>
+initialState = { currentIndex: 0 };
+
+handleClick = (index, e) => {
+  setState({
+    currentIndex: index
+  })
+}
+
+<ButtonGroup currentIndex={state.currentIndex} onIndexChange={handleClick}>
   <Button>First Button</Button>
-  <Button>Second Button</Button>
+  <Button>Second Button <span style={{
```

Comment:
could you declare this span as separate component (here, in doc file). It would be more readable that way
Is it element of design-system?

Reply:
no, this was just example for designer, removed